pkg/frontend/frontend.go

```diff
@@ -150,7 +150,7 @@ func (f *frontend) authenticatedRoutes(r *mux.Router) {
 	s.Methods(http.MethodGet).HandlerFunc(f.getAsyncOperationResult).Name("getAsyncOperationResult")
 
 	s = r.
-		Path("/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/{resourceProviderNamespace}/{resourceType}/{resourceName}/listCredentials").
+		Path("/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/{resourceProviderNamespace}/{resourceType}/{resourceName}/listcredentials").
```

Comment:
A bug it seems. Rounter can't match requests becaue of the capital `C`.

`az aro list-credentials` currently doesn't work against `master` version and the prod one.

Reply:
Just realised that we also might need to update the swagger spec and re-generate clients to include lower case URLs for consistency.

But I think this change will fix the clients even with `listCredentials` in URLs.

Reply:
it's a bug - I also spotted it this morning.  master az list-credentials doesn't work against prod because the API name completely changed (that's a separate known issue).
externally, the url path is case insensitive so the swagger is OK.  we ToLower() the path on the way in, so internally it has to be lower case to match.
need an additional unit test for this.

Reply:
issue #64